AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -220,6 +220,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79083
+mynewshub.*#%#//scriptlet("adjust-setInterval", "counter", "1500", "0.02")
```

Comment:
Why do you need `*`?

Reply:
Read @BlazDT Comments.

Reply:
Still don't get it. This `mynewsmedia.co,mynewshub.in#%#//scriptlet("adjust-setInterval", "counter", "1500", "0.02")` works ok. @BlazDT what's the point?
Btw, `/^(?!.*(bootstrapcdn.com|cdn.ampproject.org|cloudflare.com|disqus.com|disquscdn.com|dmca.com|ebacdn.com|facebook.net|fbcdn.net|fluidplayer.com|fontawesome.com|github.io|google.com|googleapis.com|googletagmanager.com|gstatic.com|jquery.com|jsdelivr.net|jwpcdn.com|jwplatform.com|polyfill.io|recaptcha.net|shrink.pe|twitter.com|ulogin.ru|unpkg.com|userapi.com|vidazoo.com|vk.com|yandex.|yastatic.net|ytimg.com|zencdn.net|player|youtube.com|cackle.me|googleoptimize.com|vuukle.com|chatango.com|twimg.com)).*$/$third-party,script,domain=mynewshub.*|mynewsmedia.*` doesn't apply on my end.

Reply:
I am not sure why, but it works on my end.

![image](https://user-images.githubusercontent.com/76908410/113863960-e4abdd80-97dc-11eb-9940-5e76a8533c25.png)

Reply:
It seems we have a bug in the extension, but it will be fixed. For now, it's better to change those domains.

Reply:
Why? I disagree.

Reply:
I've already explained it.

Reply:
I still disagree. It will still work in other platforms. When it's fixed, it will by right work itself.

Reply:
It will work only on apps with `*`. When the new version of the extension comes out, we'll come back to it.

Reply:
Why are you still changing? There are more domain i have encountered with.

Reply:
Well, add them in the next pull request.

Reply:
Read it: https://github.com/AdguardTeam/AdguardFilters/pull/79084#issuecomment-812739862. The updates in the extension is so slow, that it will take almost about 7 months for the "devs" to fix it. After the particular time, I am 100% sure that you will forget.